D2L.Hypermedia.Siren/SirenEntity.cs

```diff
@@ -39,6 +43,17 @@ public class SirenEntity : ISirenEntity {
 			m_type = type;
 		}
 
+		public SirenEntity( SerializationInfo info, StreamingContext context ) :
+			this(
+				(string[])info.GetValue( "rel", typeof(string[]) ),
+				(string[])info.GetValue( "class", typeof(string[]) ),
+				JsonConvert.DeserializeObject( info.GetString( "properties" ), typeof(ExpandoObject) ),
+				(IEnumerable<ISirenEntity>)info.GetValue( "entities", typeof(IEnumerable<ISirenEntity>) ),
+				(IEnumerable<ISirenLink>)info.GetValue( "links", typeof(IEnumerable<ISirenLink>) ),
+				(IEnumerable<ISirenAction>)info.GetValue( "actions", typeof(IEnumerable<ISirenAction>) ),
+				info.GetString( "title" ), (Uri)info.GetValue( "href", typeof(Uri) ),
+				info.GetString( "type" ) ) { }

```

Comment:
Missing `href` here ja?

Reply:
Nope I'm blind it's on the line above. In that case I rescind my above comment and replace it with a "nit: formatting" 😛